.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -47,6 +47,12 @@ in the `Generated ApiView` comment added to this PR. You can use ApiView to show
 If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
 [suppressions guide](https://aka.ms/azsdk/pr-suppressions) to get approval.
 
+</details>
+<details>
+<summary>Generating SDKs from TypeSpec</summary>
+
+For services using TypeSpec, if the Azure SDK generation fails or skips in multiple languages, check that the tspconfig.yaml is properly set up and committed in the PR. Refer to [the sample tspconfig.yaml](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml).
```

Comment:
How about using the check name directly? `Azure SDK generation` -> `"SDK azure-sdk-for-{Language}" checks`.

Reply:
This will work for data plane, but we should inform the user about the data plane one too.

Reply:
Meaning, we should add it to the data plane template too